elyra/pipeline/parser.py

```diff
@@ -185,8 +192,9 @@ def _get_input_node_ids(node_input: Dict) -> List[str]:
 
     @staticmethod
     def _get_parent_operation_links(node: Dict, embedded_node_id: Optional[str] = None) -> List[str]:
```

Comment:
The result type hint here isn't correct since it looks like this returns a list of lists of strings: `List[List[str]]`

Reply:
Good catch.... What a fine eye...